action/messageHubFeed.js

```diff
@@ -120,6 +156,20 @@ function validateParameters(rawParams) {
         return;
     }
 
+    if(rawParams.kafka_admin_url) {
+        validatedParams.kafka_admin_url = rawParams.kafka_admin_url;
```

Comment:
Update line 119 to hand the case where the body is not valid JSON. Besides that it works for me.